modules/tinymce/src/plugins/table/main/ts/api/Commands.ts

```diff
@@ -115,6 +115,27 @@ const registerCommands = (editor: Editor, actions: TableActions, cellSelection:
     mceTableDelete: eraseTable
   }, (func, name) => editor.addCommand(name, func));
 
+  Obj.each({
+    mceTableCellType: (_ui, args) => actions.setTableCellType(editor, args),
+    mceTableRowType: (_ui, args) => actions.setTableRowType(editor, args)
+  }, (func, name) => editor.addCommand(name, func));
+
+  editor.addCommand('mceTableColType', (_ui, args) =>
+    Obj.get(args, 'type').each((type) =>
+      actOnSelection(type === 'th' ? actions.makeColumnHeader : actions.unmakeColumnHeader)
+    ));
+
+  Obj.each({
```

Comment:
Given these aren't really normal commands (ie they aren't exposed via `execCommand`, should we move them to another file? eg `QueryCommands` or similar, just to make sure they are differentiated? That'd also help declutter this a bit maybe.